internal/util/openapi/openapi.go

```diff
@@ -83,5 +83,5 @@ func ConfigureOpenAPISchema(openAPISchema []byte) error {
 	// of where we got the Kubernetes openAPI schema.
 	// TODO: Refactor the openapi package in kyaml so we don't need to
 	// know the name of the kustomize asset here.
-	return openapi.AddSchema(kustomizationapi.MustAsset("openapi/kustomizationapi/swagger.json"))
+	return openapi.AddSchema(kustomizationapi.MustAsset("kustomizationapi/swagger.json"))
```

Comment:
Why is this change needed?

Reply:
Due to this change. Asset name was updated.

https://github.com/kubernetes-sigs/kustomize/commit/c1c2725360e704f4b80b7b207c0aa28a65f6eaca#diff-5a746f23b96330133cb42de78e06cfbfaf388ed252512491b44ae2d61e14a548